airflow/settings.py

```diff
@@ -68,13 +67,13 @@
 LOG_FORMAT = conf.get('core', 'log_format')
 SIMPLE_LOG_FORMAT = conf.get('core', 'simple_log_format')
 
-SQL_ALCHEMY_CONN = None  # type: Optional[str]
-DAGS_FOLDER = None  # type: Optional[str]
-PLUGINS_FOLDER = None  # type: Optional[str]
-LOGGING_CLASS_PATH = None  # type: Optional[str]
+SQL_ALCHEMY_CONN: Optional[str] = None
```

Comment:
Unrelated changes in this file now?

Reply:
I changed it because I was touching and moving PLUGINS_FOLDER and Session in other files and it felt right to correct also the other neighbouring variables. I feel that changing it to be consistent was important.